torch/csrc/jit/passes/frozen_graph_optimizations.cpp

```diff
@@ -0,0 +1,21 @@
+#include <torch/csrc/jit/passes/frozen_graph_optimizations.h>
+#include <torch/csrc/jit/ir/alias_analysis.h>
+#include <torch/csrc/jit/ir/ir_views.h>
+#include <torch/csrc/jit/passes/frozen_conv_folding.h>
+#include <torch/csrc/jit/runtime/graph_executor.h>
+#include <torch/csrc/utils/memory.h>
+
+namespace torch {
+namespace jit {
+
+void OptimizeFrozenGraph(std::shared_ptr<Graph>& graph) {
+  // run a couple times to capture Conv -> Mul -> Add etc
```

Comment:
Might be worth running that while something is changed (with some threshold to not run for too long).

Reply:
This would be nice but i'm not sure how relevant it is, might do as a follow up.

Reply:
I mentioned this simply because it's a common pattern in compilers. I agree with you that it's not clear whether it matters here.